libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -159,7 +159,7 @@ - (void)pgQuerySoup:(CDVInvokedUrlCommand *)command
         NSString *soupName = argsDict[kSoupNameArg];
         NSDictionary *querySpecDict = [argsDict nonNullObjectForKey:kQuerySpecArg];
         SFQuerySpec* querySpec = [[SFQuerySpec alloc] initWithDictionary:querySpecDict withSoupName:soupName];
-        [SFSDKHybridLogger d:[self class] format:[NSString stringWithFormat:@"pgQuerySoup with name: %@, querySpec: %@", soupName, querySpecDict]];
+        [SFSDKHybridLogger d:[self class] format:@"pgQuerySoup with name: %@, querySpec: %@", soupName, querySpecDict];
```

Comment:
This was causing a bad access exception in the tests.  I think it didn't like that there was no va_args argument in the call.

Reply:
That was probably the reason for: https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/issues/354